docs/readmes/basics/prerequisites.md

```diff
@@ -168,10 +168,7 @@ git tag -l
 
 ## Build/Deploy Tooling
 
-We support building the AGW and Orchestrator on macOS and Linux host operating
-systems. Doing so on a Windows environment should be possible but has not been
-tested. You may prefer to use a Linux virtual machine if you are on a Windows
-host.
+We support building the AGW and Orchestrator on macOS and Linux host operating systems.
```

Comment:
Should we explicitly say that Windows is not supported/tested?

Reply:
I am not sure here. The heading is ambivalent, as it talks about "Build **and** Deploy" tooling. However, the below instructions are only about deployment and not about build, afaiks.

So if we are only talking about _deployments only_, the toolchain of the five tools given below, is likely to be available on Windows. But if not properly tested by the respective developers, I would not add untested speculations to the official documentation.
However, I doubt that _building_ is possible on Windows.

So we might even want to change the title of this section to `Deployment tools`.

Thoughts?

Reply:
@themarwhal ping

Reply:
o darn I think i forgot to actually submit the comment for this. :'(

I think it's reasonable to rename this section as Deployment tools for clarity. Other than that, lgtm!

Reply:
I might lean towards explicitly noting that Windows use is not supported?

Reply:
Add-on: I can't see Scott's original comment but I agree with him that we should explicitly say what is not supported. (building on Windows).

Reply:
On second thought, the complete section about supported system is actually duplicated and should be removed, as it is clearly stated at the beginning of the document, what is supported and what not.
So I removed the unnecessary duplication such that we have a "single source of truth" [at the beginning of the document](https://docs.magmacore.org/docs/next/basics/prerequisites#operating-system).